### ISSUE TITLE
New release with PG 17 support

### ISSUE THREAD

**Issue by df7cb** [MobilityDB/MobilityDB#592]
Hi, while preparing everything in Debian for the PG 17 release, I noticed the last mobilitydb release 1.1.1 does not support it yet:
```
/home/myon/projects/postgresql/mobilitydb/mobilitydb/mobilitydb/src/general/span_analyze.c: In function ‘span_compute_stats_generic’:
/home/myon/projects/postgresql/mobilitydb/mobilitydb/mobilitydb/src/general/span_analyze.c:98:33: error: ‘VacAttrStats’ has no member named ‘attr’
   98 |   int num_hist, num_bins = stats->attr->attstattarget;
      |                                 ^~
```

It looks like it's already fixed in git (072ec587f8c67c20394aed80dfe8d21057b944f0), so a new release with this fix would be appreciated. Thanks!

Cc: @bradfordboyle

**Comment by mschoema** [comment]
Hi Christoph,
We are preparing to release both v1.1.2 and a release candidate of the upcoming v1.2.0.
Both releases will support PG 17, but v1.2 will drop support for PG 12.
I'm not sure which one you plan to use, but we will publish both in the coming days.

**Comment by bradfordboyle** [comment]
I'll plan on preparing an update package for v1.2.0 when it is released.

**Comment by mschoema** [comment]
We just released both v1.1.2 and v1.2.0rc1, both supporting PG 17.
We will release v1.2.0 in the coming week, unless someone detects big issues with the release candidate.
Let us know if we can help in any way concerning the packaging.

**Comment by mschoema** [comment]
MobilityDB 1.2.0 was just released: https://github.com/MobilityDB/MobilityDB/releases/tag/v1.2.0
Please let us know if there are any issues with it or if we can help with the packaging in any way.
